.evergreen/serverless/create-instance.sh

```diff
@@ -46,6 +46,17 @@ if [ -z "$SERVERLESS_INSTANCE_NAME" ]; then
     SERVERLESS_INSTANCE_NAME="$RANDOM-DRIVERTEST"
 fi
 
+# Ensure that a Python binary is available for JSON decoding
+# shellcheck source=.evergreen/find-python3.sh
+. ../find-python3.sh || return
+
+PYTHON_BINARY="$(find_python3)"
+
+if [ -z "$PYTHON_BINARY" ]; then
+    echo "Failed to find Python3 binary"
+    exit 1
+fi
```

Comment:
I believe this can also be written as:
```bash
PYTHON_BINARY="$(find_python3)" || exit 1
```
Since `find_python3` already emits messages indicating when it is unable to find a binary.

Reply:
So long as I don't redirect stderr to `/dev/null`, that works we'll see output like the following:

> Could not find a python3 binary capable of creating a virtual environment!

I wasn't sure how this would work in practice, but https://stackoverflow.com/a/17336953/162228 explained the difference between `find_python3`'s exit code and the stdout output that we're capturing here. So `||` in your suggestion is actually branching on whether an empty string was returned, which is equivalent to the more verbose conditional I had here.

Thanks for the suggestion! I'll make that change.